WcaOnRails/app/views/devise/registrations/new.html.erb

```diff
@@ -46,7 +46,7 @@
             <%= f.input :name %>
             <%= f.input :dob, as: :date_picker %>
             <%= f.input :gender, collection: User::ALLOWABLE_GENDERS, label_method: lambda { |g| { m: t('wca.devise.gender.male'), f: t('wca.devise.gender.female'), o: t('wca.devise.gender.other_gender') }[g] } %>
-            <%= f.input :country_iso2, collection: Country.real, value_method: lambda { |c| c.iso2 }, label_method: lambda { |c| c.name } %>
+            <%= f.input :country_iso2, collection: Country::ALL_COUNTRIES_BY_LOCALE[I18n.locale].select(&:real?), value_method: lambda { |c| c.iso2 }, label_method: lambda { |c| c.name } %>
```

Comment:
I'd make a method hiding some of these unseemly details. Could look like this:

`Country::all_sorted_by(I18n.locale, real: true), value_method: ...`